Sources/PortalSwift/Mocks/Constants.swift

```diff
@@ -36,6 +36,7 @@ public let mockDecryptResult = "{\"data\":{\"plaintext\":\"signingShareObject\"}
 public let mockEncryptResult = "{\"data\":{\"key\":\"someKey\",\"cipherText\":\"\(mockCiphertext)\"},\"error\":{\"code\":0,\"message\":\"\"}}"
 
 public let mockEncryptWithPasswordResult = "{\"data\":{\"cipherText\":\"\(mockCiphertext)\"},\"error\":{\"code\":0,\"message\":\"\"}}"
+public let mockEjectWalletAndDiscontinueMPC = "{\"privateKey\":\"099cabf8c65c81e629d59e72f04a549aafa531329e25685a5b8762b926597209\",\"error\":{\"code\":0,\"message\":\"\"}}"
```

Comment:
if this is the case then yea you need to add error checking after you call eject from the binary

Reply:
Addressed